### PR TITLE
unravel 0.2.2 (new formula)

### PR DESCRIPTION
**Pull request by bfontaine**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/unravel.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  url "https://github.com/pesterhazy/unravel/archive/0.2.2.tar.gz"
+  sha256 "144b2a96dc5900dada9e668f765ac63d55b97770525f1c994b8b41084048a9cc"
+
+  devel do
+    url "https://github.com/Unrepl/unravel/archive/v0.3.0-beta.2.tar.gz"
+    sha256 "786acffe0a2b0dc7fb2675215764739fee39a5da1b9ac72d7c8c49afab785408"
+  end
+
+  depends_on "node"
+  depends_on "lumo"
```
Comment (ilovezfs) [inline Formula/unravel.rb]: this isn't build time only?

File: Formula/unravel.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  def install
+    # This is installed by Homebrew
+    lumo_version = build.stable? ? "1.6.0" : "1.7.0"
```
Comment (ilovezfs) [inline Formula/unravel.rb]: not sure I understand this. why are we having to hard-code version numbers?

File: Formula/unravel.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      '"${UNRAVEL_HOME}/node_modules/lumo-cljs/bin/lumo"', "lumo"
+
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+
+    (bin/"unravel").write_env_script libexec/"bin/unravel",
+      :UNRAVEL_HOME => libexec/"lib/node_modules/unravel-repl"
+  end
+
+  test do
+    # We'd need a REPL to connect to in order to have a proper test
```
Comment (ilovezfs) [inline Formula/unravel.rb]: You can assert a failure instead.

File: Formula/unravel.rb (hunk 2 above)
Comment (bfontaine) [reply Formula/unravel.rb]: This is for the pattern on the line below. We could use something like `/"lumo-cljs": ".*?",/` instead, if you prefer.

File: Formula/unravel.rb (hunk 1 above)
Comment (bfontaine) [reply Formula/unravel.rb]: [Nope](https://github.com/Unrepl/unravel/blob/f18e7ad263931b76fe5c3ac2546e293695a58914/bin/unravel#L20).

Comment (ilovezfs) [reply Formula/unravel.rb]: let's alphabetize these

File: Formula/unravel.rb (hunk 2 above)
Comment (ilovezfs) [reply Formula/unravel.rb]: yeah that seems better

File: Formula/unravel.rb (hunk 3 above)
Comment (bfontaine) [reply Formula/unravel.rb]: Done.

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
Thanks @bfontaine!

**Pull request closed without merging**